### PR TITLE
Allows HUDs to be emagged with an emag as well as an EMP

### PR DESCRIPTION
**Pull request by Shadowlight213**
Not sure why this was supposed to be a thing, with how rare EMPs are on the station. Emagging a sechud will allow you to remove any id locks on it and modify sec records without needing to get sec access on your id 

I basically made this off of the commented out code in #560, so feedback is appreciated.

### CODE REVIEW

**Comment by Man-Shroom** [comment]
Booo, muh ghetto and changeling sechuds.

It should work off of both emag and EMP.

**Comment by Tokiko1** [comment]
I agree with Man-Shroom. We need more items that are both EMP and emagable so both changelings and traitors can use them.

**Comment by palpatine213** [comment]
Alright. I see no reason not too.

**Comment by KorPhaeron** [comment]
Why not both?

**Comment by xxalpha** [comment]
Don't forget the implants.

**Comment by palpatine213** [comment]
@xxalpha From what I can tell, sechud implants currently don't support emagging at all.

**Comment by xxalpha** [comment]
Well you are adding the option for sec huds to be emagged (I assume they aren't hackable yet?), so maybe you should add that option to sec huds for the sake of consistency.

Also you should be using emag_act().

**Comment by palpatine213** [comment]
The ability for sechuds to have their ID lock removed has always existed. It just required an EMP rather than an emag. I think sec record consoles need an emp to mess with them as well.
I'll switch to emag_act ().
I can try to add the functionality to implants, but I have no clue how to do that.

**Comment by xxalpha** [comment]
>It just required an EMP rather than an emag.

Wow that's stupid.

Now that I think about it, don't touch the implants. I'd rather do it myself than deal with conflicts.

**Comment by palpatine213** [comment]
Yeah, the EMP thing is dumb. It's probably one of the closest things we have to a secret, since its so damn obscure. 
I was originally planning on making it emag only, but it was pointed out that if it was both, lings would be able to benefit from the functionality as well, since they have that emp screech ability.

File: code/modules/clothing/glasses/hud.dm [hunk 1]
```diff
@@ -65,6 +73,13 @@
 		emagged = 1
 		desc = desc + " The display flickers slightly."
 
+/obj/item/clothing/glasses/hud/security/sunglasses/emag_act(mob/user)
```
Comment (Cheridan) [inline code/modules/clothing/glasses/hud.dm]: This is handled by the /obj/item/clothing/glasses/hud/emag_act() already so it shouldn't be needed.

**Pull request merged**